src/actions/application.js

```diff
@@ -35,6 +35,11 @@ function* loadMenu() {
     yield put(resetLoading());
     yield put(showLoading());
     const menuLinks = yield call(api, 'menu');
+    if (menuLinks.message) {
```

Comment:
I'm wondering whether we could look into the cookie and see whether the resulting response had one. ❓